cli/__tests__/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
To get around the no-global-require rule tripping since those would be in basically every test.